sources/install/package_ad.sh

```diff
@@ -718,6 +719,16 @@ function install_pcredz() {
     add-to-list "pcredz,https://github.com/lgandx/PCredz,PowerShell credential dumper"
 }
 
+function install_spray() {
+  # Password Spraying tool for Active Directory Credentials.
+  colorecho "Installing Spray"
+  git -C /opt/tools/ clone https://github.com/Greenwolf/Spray.git
```

Comment:
Can you add `--depth=1` after the `clone` so that it doesn't download everything ?